.github/workflows/tests.yml

```diff
@@ -39,8 +39,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
-        os: [ubuntu-latest, windows-latest, macos-latest]
+        python-version: ["3.8"]
+        os: [ubuntu-latest]
```

Comment:
An earlier commit had python 3.8-3.12 and also had mac and windows OS. Unfortunately, it looks like lolo is not up to date so I reverted back to ubuntu and 3.8. I plan to make another PR to bring lolo into the present